src/docs/release/breaking-changes/enterText-trailing-caret.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: Change the enterText method to move the caret to the end of the input text
-description: `WidgetTester.enterText` and `TestTextInput.enterText` now move the caret to the end of the input text.
+description: WidgetTester.enterText and TestTextInput.enterText now move the caret to the end of the input text.
```

Comment:
This is the build error I assume?

Reply:
Oh, I just realized what you meant. Yes.  :D